test/eth.sendTransaction.js

```diff
@@ -37,7 +49,9 @@ var tests = [{
     call: 'eth_'+ method
 },
 // test with gasPrice missing
-{
+// This test is broken but very difficult to diagnose cause....
+// behavior is correct (see sendTransaction E2E)
+/*{
```

Comment:
I [broke this test]https://travis-ci.org/ethereum/web3.js/jobs/599482864#L3763 somehow and cannot figure out how to fix it. Very frustrating. It does not actually test any code in eth-accounts - I think the problem might be something unusual about the way this test file is structured....

The test itself checks whether or not `gasPrice` is fetched when calling `eth.sendTransaction`, so I have explicitly added that case in`e2e.methods.send` in this PR. 

idk, ggrrrrrr.

Reply:
Fixed - this was some kind of problem using _.clone. ...